pybv/tests/test_bv_writer.py

```diff
@@ -137,3 +133,16 @@ def test_scale_data():
     data_written = np.fromfile(tmpdir + '/' + fname + '.eeg', dtype=np.float32)
     assert_allclose(data_written, data.T.flatten())
     rmtree(tmpdir)
+
+
+@pytest.mark.parametrize("resolution", np.logspace(-3, -9, 7))
+@pytest.mark.parametrize("unit", ["V", "mV", "uV", "µV", "nV"])
```

Comment:
Why `np.logspace(-3, -9, 7)`? It is of len==7, while the unit list is of len==5

Reply:
I think it's the cartesian product so the test will be run 7*5=35 times

Reply:
Yep.